Marlin/src/lcd/language/language_it.h

```diff
@@ -191,8 +191,8 @@ namespace Language_it {
   LSTR MSG_IDEX_MODE_MIRRORED_COPY        = _UxGT("Copia speculare");
   LSTR MSG_IDEX_MODE_FULL_CTRL            = _UxGT("Pieno controllo");
   LSTR MSG_IDEX_DUPE_GAP                  = _UxGT("X-Gap-X duplicato");
-  LSTR MSG_HOTEND_OFFSET_Z                = _UxGT("2° ugello Z");
-  LSTR MSG_HOTEND_OFFSET_A                = _UxGT("2° ugello @");
+  LSTR MSG_HOTEND_OFFSET_Z                = _UxGT("2") LCD_STR_DEGREE _UxGT(" ugello Z");
```

Comment:
° is used as short form of second 'nd', not degree

Reply:
fixed